cmake/modules/AddSwift.cmake

```diff
@@ -630,13 +630,6 @@ function(_add_swift_lipo_target)
         CUSTOM_TARGET_NAME "${LIPO_TARGET}"
         OUTPUT "${LIPO_OUTPUT}"
         DEPENDS ${source_targets})
-  else()
-    # We don't know how to create fat binaries for other platforms.
-    add_custom_command_target(unused_var
-        COMMAND "${CMAKE_COMMAND}" "-E" "copy" "${source_binaries}" "${LIPO_OUTPUT}"
-        CUSTOM_TARGET_NAME "${LIPO_TARGET}"
-        OUTPUT "${LIPO_OUTPUT}"
-        DEPENDS ${source_targets})
```

Comment:
Minor: One thing I did here is adding a `precondition`. If someone were to add a new call to `_add_swift_lipo_target`, CMake will refuse to generate for any SDK which is not Darwin.